.github/workflows/integration_tests.yml

```diff
@@ -898,162 +836,6 @@ jobs:
         with:
           filename: integration-tests-failed.md
 
-      - name: Find PR number for current build
```

Comment:
All the code related to creating a draft release was remove since this now lives in the `release.yml` and `prerelease.yml` pipelines.